src/appendices/phenotype.md

```diff
@@ -0,0 +1,250 @@
+# Tabular phenotypic data guidelines
+
+This appendix is a collection of guidelines and examples utilizing these guidelines for creating well-organized aggregated tabular phenotypic data.
+
+## Guidelines
+
+The following guidelines are all **RECOMMENDED** when preparing tabular phenotypic data like the participants file, sessions file, demographics file, or phenotypic and assessment data. The language below uses REQUIRED, MUST, and others to imply these are the requirements for these **RECOMMENDED** guidelines.
+
+### 1. Aggregation
+
+Tabular phenotypic data files are prepared as one pair of data dictionary JavaScript Object Notation (JSON) file and data tab-separated value (TSV) file. Aggregation refers to the contents of the TSV file, collecting all participant data into one TSV per tabular phenotypic file.
+
+### 2. Phenotypic and assessment data
+
+In phenotypic and assessment data each measurement tool has an independent aggregated data TSV file in which the user collects all subjects, sessions, and/or runs of data as one entry per row (with a row defined by the smallest unit of acquisition). In other words:
+
+1. Each row MUST start with `participant_id`.
+2. Each TSV file MUST contain a `session_id` column when multiple [sessions](../glossary.md#session-entities)[^1] are present in the data set regardless of whether those sessions are in the `phenotype/` data, `sub-<label>/` data, or a combination of the two.
+3. If more than one of the same measurement tool is acquired within the same `session_id`, a run column MUST be added.
+4. To encode the acquisition time for a measurement tool’s `session_id`, add the `session_id` to the sessions file and include the OPTIONAL `acq_time` column.
+
+Furthermore, if you have to add a `session_id` column to the tabular phenotypic data, you then MUST also introduce a session directory to the imaging data, even if only one imaging session has been created. This rule can be considered as "**if anyone uses sessions, everyone uses sessions**." And vice versa, if imaging data has session directories, all imaging data and tabular phenotypic data MUST have sessions.
+
+This produces a file in which same-participant entries can take up as many rows as needed according to the smallest unit of acquisition.
+
+### 3. Sessions file
+
+If there is more than one session for any one participant, then it is REQUIRED to provide a sessions file at the dataset root. The sessions file MUST list all sessions for all subjects across imaging and tabular phenotypic data.
+
+When a sessions file is in use, you MUST NOT provide participant-level sessions files which would otherwise use the inheritance principle. If a sessions file is provided, then it MUST begin with a `participant_id` column followed immediately by a `session_id` column. The data dictionary JSON’s `session_id` field MUST include `Levels` with the description of each possible `session_id`.
```

Comment:
```suggestion
When a sessions file is in use, you MUST NOT provide additional sessions files at the  participant-level which would otherwise use the inheritance principle. If a sessions file is provided, then it MUST begin with a `participant_id` column followed immediately by a `session_id` column. The data dictionary JSON’s `session_id` field MUST include `Levels` with the description of each `session_id`.
```
I think "possible" implies "future" but I think you mean "current" or "present"